docs/blog/posts/release0-23-0.md

```diff
@@ -0,0 +1,87 @@
+---
+draft: false
+date: 2023-05-03
+categories:
+  - Release
+title: "Textual 0.23.0 improves message handling"
+authors:
+  - willmcgugan
+---
+
+# Textual 0.23.0 improves message handling
+
+It's been a busy couple of weeks at Textualize.
+We've been building apps with [Textual](https://github.com/Textualize/textual), as part of our *dog-fooding* week.
+The first app, [Frogmouth](https://github.com/Textualize/frogmouth), was released at the weekend and already has 1K GitHub stars!
+Expect two more such apps this month.
+
+<!-- more -->
+
+<div>
+--8<-- "docs/blog/images/frogmouth.svg"
+</div>
+
+!!! tip
+
+    Join our [mailing list](http://eepurl.com/hL0BF1) if you would like to be the first to hear about our apps.
+
+We haven't stopped developing Textual in that time.
+Today we released version 0.23.0 which has a really interesting API update I'd like to introduce.
+
+Textual *widgets* can send messages to each other.
+To respond to those messages, you implement a message handler with a naming convention.
+For instance, the [Button](/widget_gallery/#button) widget sends a `Pressed` event.
+To handle that event, you implement a method called `on_button_pressed`.
+
+Simple enough, but that method will be called to handle *all* pressed events.
+If you have multiple buttons you typically ended up writing a large `if` statement.
```

Comment:
Tense gets a little twisted here. I'd go with either "you *had* multiple" or "*end* up writing"; something like that.